src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/Microsoft.NET.Sdk.Functions.Version.props

```diff
@@ -11,7 +11,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <FunctionsSdkVersion>3.0.8</FunctionsSdkVersion>
+    <FunctionsSdkVersion>3.0.9</FunctionsSdkVersion>
```

Comment:
Why do we need this?

Reply:
For the next set of changes since 3.0.8 is already released. I have a change lined up but wanted the version update to be a separate PR so that we know where the changes for 3.0.9 starts.